tests/dummy/app/router.js

```diff
@@ -1,12 +1,13 @@
 import Ember from 'ember'
+const {Router} = Ember
```

Comment:
import config from './config/environment'
import Ember from 'ember'
const {Router: EmberRouter} = Ember

const Router = EmberRouter.extend({
  location: config.locationType
})

Router.map(function () {
  this.route('demo', {path: '/'})
})

export default Router